docs/modules/ROOT/pages/references/parameters.adoc

```diff
@@ -0,0 +1,41 @@
+= Parameters
+
+The parent key for all of the following parameters is `minio`.
+
+
+== `namespace`
+
+[horizontal]
+type:: string
+default:: `syn-minio`
+
+
+
+== `accessKey`
```

Comment:
Parameters documentation should be updated to match new parameters structure in `class/defaults.yml`.

Reply:
Done